### DISCUSSION TITLE
Updates to "fsprojects" policies

### DISCUSSION THREAD

**Discussion by dsyme** [fsprojects/FsProjectsAdmin#29] [General]
@sergey-tihon  and I have updated the policies we use for "fsprojects" to clarify things like when we would consider archiving a project, see https://github.com/fsprojects/fsprojects-admin/blob/master/README.md

If you'd like to check the policies and/or comment please discuss here.

To follow future policy updates pleas watch this repo.

**Comment by dsyme** [comment]
cc @fsprojects/aws-team @fsprojects/azure-team @fsprojects/gui-team @fsprojects/paket-team @fsprojects/2fa-now-required-for-fsprojects-members-maintainers-please-enable-thanks @fsprojects/collections-team @fsprojects/excel-team @fsprojects/fleece-team @fsprojects/fsharplus-team @fsprojects/fsharpx-team @fsprojects/typeprovider-authors @fsprojects/typeprovider-sdk-owners @fsprojects/tooling-team

**Comment by nojaf** [comment]
This seems reasonable to me.

**Comment by jwosty** [comment]
These seem reasonable. The only one that worries me is the "two maintainers" policy. I understand why, but I'm the sole contributor to fsprojects/Interstellar . However if this is grounds to remove from fsprojects, I can definitely understand why (perhaps it simply doesn't have enough actual usage outside my own purposes).

**Reply by wallymathieu** [reply]
I agree. Perhaps someone is interested in starting to work together on your project?

**Reply by sergey-tihon** [reply]
> However if this is grounds to remove from fsprojects, I can definitely understand why (perhaps it simply doesn't have enough actual usage outside my own purposes).

Two maintainers per projects is our goal (for sustainability). We never removed active projects from organisation because due to this  policy and do not plan to do so. 

@dsyme  correct me if I am wrong.

**Reply by wallymathieu** [reply]
One of the big reasons why I would want code on fsprojects instead of on my personal github is due to the implication that someone might be able to take over if anything happens. Say that I'm hospitalized and unable to fix some serious issue, but someone else could fix it and publish it to NuGet. The fsprojects community seems to be better at keeping some projects alive (since there are more people and more of them are engaged in the community).

**Reply by wallymathieu** [reply]
So that's assuming that a project actually gives someone else value enough that it makes sense for someone else to contribute.

**Reply by dsyme** [reply]
> @dsyme correct me if I am wrong.

You're definitely correct! :)

**Reply by jwosty** [reply]
> I agree. Perhaps someone is interested in starting to work together on your project?

It mostly grew out of a personal project that's currently actively using it, so even if nobody uses it, I don't mind since I'm getting good use out of it myself. Either way, I would welcome anyone who wants to contribute! [fsprojects/Interstellar](https://github.com/fsprojects/Interstellar)

**Comment by dsyme** [comment]
> These seem reasonable. The only one that worries me is the "two maintainers" policy. 

TBH it's listed in the policies as an aspiration not a requirement. Several projects have zero maintainers (abandoned, will be archived eventually), many have one, some have multiple

**Reply by jwosty** [reply]
Alright, LGTM then!
